doc/package-notes.xml

```diff
@@ -814,4 +814,46 @@ citrix_receiver.override {
    </para>
   </section>
  </section>
+ <section xml:id="sec-ibus-typing-booster">
+  <title>ibus-engines.typing-booster</title>
+
+  <para>This package is an ibus-based completion method to speedup typing.</para>
+
+  <section xml:id="sec-ibus-typing-booster-activate">
+   <title>Activate engine</title>
```

Comment:
Try using “Activating the engine” or “Enabling the engine”, imperative form does not really suit titles.